e2e/Makefile

```diff
@@ -1,20 +1,25 @@
 export GO111MODULE=on
 
-IMAGE_VERSION=canary
-K8S_VERSION?=""
-GINKGO_PATH := $(shell go env GOPATH)/bin/ginkgo
+IMAGE_VERSION?=canary
+K8S_VERSION?="v1.28.7"
 
-.PHONY: test clean check-token install-terraform test-manifest
+.PHONY: test clean check-token test-manifest
 
-test-manifest:
+test-manifest: kustomize
 	../hack/generate-yaml.sh $(IMAGE_VERSION) > test/manifest/linode-blockstorage-csi-driver.yaml
 
-${GINKGO_PATH}:
-	go install github.com/onsi/ginkgo/v2/ginkgo@v2.13.2
+test-cluster: ctlptl clean-cluster
```

Comment:
Would it make sense to make `test-manifest` a dependency of `test-cluster` since `linode-blockstorage-csi-driver.yaml` is consumed [here](https://github.com/linode/linode-blockstorage-csi-driver/pull/163/files#diff-0908e8aad04dc51bce3a028b466879848a0cf5e57f2ee1af8bf08a599c600e23R24)?